vsintegration/src/FSharp.Editor/Refactor/ChangeTypeofWithNameToNameofExpression.fs

```diff
@@ -27,6 +28,8 @@ type internal FSharpChangeTypeofWithNameToNameofExpressionRefactoring [<Importin
 
             let! parseResults =
                 document.GetFSharpParseResultsAsync(nameof (FSharpChangeTypeofWithNameToNameofExpressionRefactoring))
+                |> CancellableTask.start context.CancellationToken
```

Comment:
Just curious, does this still add a perf benefit in the current shape? I mean two extra operations with conversions.

Reply:
Maybe not in asyncMaybe, but no other way of running ctask from it.

Reply:
Alternative would be to leave the method async/asyncMaybe and revert majority of this pr